README.md

```diff
@@ -60,27 +60,51 @@ For first-time jspm users, you will need to configure it to use GitHub:
 
 Follow the steps to enable GitHub access.
 
-### Getting started
+### Getting started: docker development
+Clone this repository.
+
+Create the file `/etc/resolver/dev` with the following contents (requires root permission):
+
+```
+nameserver 127.0.0.1
+port 53535
+```
+
+Run:
+- `cd example`
+- `docker-compose up`
+- Open `https://www.example.dev:3000`
+
```

Comment:
Can you add a step linking to 'Accepting the development SSL certificate` below?